src/App.js

```diff
@@ -142,7 +142,8 @@ function HistoryGraph({ countryData }) {
       animationEnabled: true,
       height: 260,
       title: {
-        text: "Trends for COVID-19 Cases in Australia ",
+        text: "Overall trends for COVID-19 cases in Australia ",
+        fontFamily: "Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif",
```

Comment:
why we need this long fontFamily?

Reply:
This is the same font family used for the headings on the rest of the page. It doesn't need to be that long, but they're all quite good backup fonts so I wasn't inclined to change it.